### DISCUSSION TITLE
Detail view not working - Solved

### DISCUSSION THREAD

**Discussion by jorgevespa** [Power-Components/livewire-powergrid#721] [Q&A]
Hello, first let me tell you that you are doing an amazing job here.

I have been working with the powergrid datatable using the documentation, which is very complete.

I am trying to use the "detail row" option but with no luck:

i have added the "detail" feature in the setUp function:

```
Detail::make()
                ->view('components.detail')
                ->options(['menssage' => 'hello world'])
                ->showCollapseIcon(),
```

I have created the folder "components" in the /resources/views/ folder, because it was not there, and created a file: detail.blade.php with:

```
<div class="p-2 bg-white border border-slate-200">
    <div>Id {{ $id }}</div>
    <div>Options @json($options)</div>
    <div>Nombre {{ $row->nombre_canal }}</div>
</div>
```

The "show detail icon" appears but when i click on it, it doesnt show anything and also no errors on the console or log (instaled debugbar also).

I am using the latest version of laravel and the component.

Please help me, what i could be doing wrong or where i can look more to solve it?

Thanks.

A capture of the grid/button working, but not displaying the view:
![Screenshot_1](https://user-images.githubusercontent.com/46542787/201827511-000b2a1d-9d78-41e5-ac0d-ce5169744796.png)

**Comment by luanfreitasdev** [comment]
Hello!. That's weird, I've never heard anything like it. Do you know if in some previous version this would work for you?

**Reply by jorgevespa** [reply]
Hello, thanks for your time, it is my first attempt using powergrid.
Today i was doing some more test.

Install the demo repository from Git, wihch is working ok with the "detail view", there is no error.

So install an absolutely clean laravel+jetstream+livewire+powergrid instance, install the demo as explained in the documentation, apply little changes to the demo so it can show use the "detail view" of users model and no luck again, the only weird i can see is the error in the screen bellow:
![Screenshot_2](https://user-images.githubusercontent.com/46542787/202072143-23800201-88f0-4903-b261-1de638dfdfa0.png)

I hope you can help me to understand the problem.

Thanks.

**Comment by DZ4-Miguel** [comment]
Hi, I gave the same issue. Did u solve the problem?

**Reply by jorgevespa** [reply]
Hi, unfortunately no, i have tried but not success.

**Comment by ultimatee** [comment]
Well, I have the same exact issue. Did any of you solved it in the meantime?

When the arrow is clicked, there is a POST request sent to the server in the form of
http://servername/livewire/message/powergrid-table-name 
Should the server respond with something special?

**Reply by ultimatee** [reply]
Apparently I had no alpineJS included.
When I included
`<script src="//unpkg.com/alpinejs" defer></script>`
the detail row was shown.

**Reply by jorgevespa** [reply]
Hello, you are right, it looks like it is necessary, for others having the same issue, more detailed:

Edit: /resources/views/layouts/app.blade.php

Add alpineJS after the @scripts as mentioned @ultimatee :
        @livewireScripts
        @powerGridScripts
        <script src="//unpkg.com/alpinejs" defer></script>
    </body>

Thanks.
